modules/media/help/edit.md

```diff
@@ -0,0 +1,3 @@
+# Edit Media File
+This page allows users to modify information about an uploaded file.  Fields such as Date of Administration, Comments, and file visibility (hidden or visible) can be updated.  Note that fields such as PSCID, Visit Label, Site, and Instrument cannot be modified on uploaded media files. 

```

Comment:
Site and Instrument aren't proper nouns, they should be lower case.

Reply:
They're capitalized here to reflect the field labels printed in the form.

Reply:
In that case they should have quotation marks.